bedrock/settings/base.py

```diff
@@ -511,7 +506,6 @@ def set_whitenoise_headers(headers, path, url):
     '/contribute/',
     '/about/',
     '/contact/',
-    '/thunderbird/',
```

Comment:
There's a comment saying this list must match a list at CloudFlare. What do we need to do to make that happen?

Reply:
This is probably a question for @jgmize

Reply:
That just means that the /thunderbird/ without locales (so the redirects) URLs are allowed to be cached with a "Vary: Accept-Language" header. It should be safe to remove this and we can remove the /thunderbird config from CloudFlare later.

Reply:
👍